CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 Version XXX:
 
+* updated root certificates in examples
```

Comment:
Should read "update" not "updated"

Reply:
Wasn't resolved as far as I can tell, it still says "updated"

Reply:
it was resolved in my local repo until I messed up the rebase -i.
I am doing it again.

Reply:
ok, let's try that again